src/developTest/java/mServer/crawler/AddToFilmlistTest.java

```diff
@@ -0,0 +1,123 @@
+package mServer.crawler;
+
+import java.io.IOException;
+import mSearch.daten.DatenFilm;
+import mSearch.daten.ListeFilme;
+import okhttp3.mockwebserver.*;
+
+import org.junit.*;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.junit.Assert.*;
+
+public class AddToFilmlistTest {
+    private static final String FILM_NAME_ONLINE = "onlinefilm.mp4";
+    private static final String FILM_NAME_ONLINE2 = "onlinefilm2.mp4";
+    
+    private static MockWebServer mockServer;
+    private static String baseUrl;
+    
+    private ListeFilme list;
+    private ListeFilme listToAdd;
+    
+    @BeforeClass
+    public static void setUpClass() throws IOException {
+        mockServer = new MockWebServer();
+        Dispatcher dispatcher = new Dispatcher() {
+
+            @Override
+            public MockResponse dispatch(RecordedRequest request) throws InterruptedException {
+
+                switch(request.getPath()) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Add a default case to this switch. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this "switch" statement by "if" statements to increase readability. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1301)